lib/index.ts

```diff
@@ -23,6 +24,16 @@ function inspect(root: string, targetFile?: string, options?: any) {
         manifestExcludeGlobs: options.manifestExcludeGlobs,
       }
     : {};
+
+  const staticScanningOptions: StaticScanningOptions =
+    options && options.staticScanningOptions
+      ? {
+          imagePath: options.staticScanningOptions.imagePath,
```

Comment:
Not sure why these are not part of the dockerOptions?

Reply:
Because they are not Docker options  😄 Arguably we've opted to name them "Docker options" whereas they are just any key-value options, and seeing that they're currently used with Docker flags in mind, I opted not to pollute the interface

Reply:
That makes sense. The name `dockerOptions` is a bit of a misnomer here and yes I would agree with @jurglic - let's handle this then in a separate function and not pollute inspect().

Reply:
Thanks! Will do that as part of a separate PR!